CRM/Contact/Page/AJAX.php

```diff
@@ -786,29 +786,29 @@ public static function getDedupes() {
 
     $count = 0;
     foreach ($dupePairs as $key => $pairInfo) {
-      $pair =& $pairInfo['data'];
+      $pair = $pairInfo['data'];
```

Comment:
Agreed no point passing by reference, I can't see anything being changed. But actually let's lose this variable altogether. $pair is only used 4 ish times in the foreach loop, so we can write out $pairInfo['data'] longhand. This is done several times in the loop anyway, so it's unnecessary. Unless you can spot a use for it.